src/app/public/modules/tabs/tab.component.ts

```diff
@@ -35,13 +35,17 @@ export class SkyTabComponent implements OnDestroy, OnChanges {
   @Input()
   public active: boolean;
 
+  /**
+   * A lower-case string to represent this tab in the URL permalink.
+   * Set to `undefined` to automatically generate a name from the `tabHeading` property.
+   */
   @Input()
-  public set queryParamValue(value: string) {
-    this._queryParamValue = this.sanitizeKey(value);
+  public set permalinkValue(value: string) {
+    this._permalinkValue = this.sanitizeName(value);
   }
 
-  public get queryParamValue(): string {
-    return this._queryParamValue || this.sanitizeKey(this.tabHeading);
+  public get permalinkValue(): string {
+    return this._permalinkValue || this.sanitizeName(this.tabHeading);
```

Comment:
What happens if `tabHeading` is undefined? Should we account for that?

Reply:
The input `tabHeading` is listed as a required field:
https://developer.blackbaud.com/skyux/components/tabs#tab-properties